src/utils/queries.js

```diff
@@ -0,0 +1,38 @@
+export const createAgentTable = `
+DROP TABLE IF EXISTS agents;
+CREATE TABLE "agents" (
+  "id" serial primary key,
+  "first_name" varchar(255) NOT NULL,
+  "last_name" varchar(255) NOT NULL,
+  "email" varchar UNIQUE NOT NULL,
+  "password" varchar NOT NULL,
```

Comment:
what's the `varchar` length? shouldn't this be `varchar(255)`? please update all references